src/game/features/self/SuperDamage.cpp

```diff
@@ -0,0 +1,30 @@
+#include "core/commands/LoopedCommand.hpp"
+#include "game/features/Features.hpp"
+#include "game/rdr/Enums.hpp"
+#include "game/rdr/Natives.hpp"
+
+namespace YimMenu::Features
+{
+	class SuperDamage : public LoopedCommand
+	{
+		using LoopedCommand::LoopedCommand;
+
+		virtual void OnTick() override
+		{
+				const float DAMAGE_MULT = 100000000.0;
```

Comment:
If there is care for however high this is, you can use `std::numeric_limits<float>::max();`

Reply:
eventually im would like to rework this as an input value or slider